hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/TableResolverImpl.java

```diff
@@ -148,19 +149,21 @@ private Mapping resolveMapping(Mapping mapping) {
         } else {
             connector = connectorCache.forType(type);
         }
+        String objectType = mapping.objectType() == null ? connector.defaultObjectType() : mapping.objectType();
```

Comment:
Do we have this covered by tests in sql module? not by mongo connector.

Reply:
[CR comments](https://github.com/hazelcast/hazelcast/pull/24356/commits/b4df8f0363a24a35130ecd89d697105fb5413292)